sdk/communication/azure-communication-identity/azure/communication/identity/_communication_identity_client.py

```diff
@@ -188,20 +188,30 @@ def revoke_tokens(
     @distributed_trace
     def get_token_for_teams_user(
             self,
-            add_token, # type: str
+            aad_token, # type: str
+            app_id, # type: str
+            user_id, # type: str
```

Comment:
Just a note: I think we want to keep these expanded and not wrapped in an options bag as on other platforms. It's not idiomatic for Python to have options bags.